taskcluster/mozillavpn_taskgraph/transforms/beetmover.py

```diff
@@ -0,0 +1,108 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+
+import os.path
+
+from taskgraph.transforms.base import TransformSequence
+from taskgraph.util.schema import resolve_keyed_by
+
+transforms = TransformSequence()
+
+
+@transforms.add
+def add_beetmover_worker_config(config, tasks):
+    for task in tasks:
+        worker_type = task["worker-type"]
+        task_name = task["name"]
+        task_label = f"{worker_type}-{task_name}"
+        resolve_keyed_by(
+            task,
+            "bucket",
+            item_name=task_label,
+            **{"level": config.params["level"]},
+        )
+        bucket = task["bucket"]
+        # When we add balrog releases we will need to add an action to beetmoverscript
+        action = "push-to-nightly"
```

Comment:
In `beetmoverscript` it looks like `push-to-candidates` is an alias for `push-to-nightly`. I propose we use that string here as it matches closer to reality what is actually happening.

Reply:
Nice, didn't realize we had that alias 👍🏼